README.md

```diff
@@ -67,6 +67,19 @@ foreach ($items as $item) {
 
 In the code, `<ul />` creates a ul with no children. Then we dynamically append children to it for each item in the `$items` list.
 
+Alternatively, you can pass an array of children to `<ul>...</ul>` instead. This is especially useful when dealing with larger xhp trees where it would be harder to get a variable reference to the `ul`.
```

Comment:
```suggestion
Alternatively, you can pass a `vec` of children to `<ul>...</ul>` instead. This is especially useful when dealing with larger xhp trees where it would be harder to get a variable reference to the `ul`.
```

Reply:
I intentionally used the word `array` here. Xhp-lib can handle all Hack arrays in hhvm 4.103+ and all legacy arrays in hhvm 4.102-. AnyArray implements XhpChild.

https://github.com/facebook/hhvm/blob/master/hphp/hack/hhi/hackarray.hhi#L19

```HACK
abstract class AnyArray<<<__RequireDynamic>> +Tk as arraykey, <<__RequireDynamic>> +Tv>
implements KeyedContainer<Tk, Tv>, \XHPChild {}
```